src/audio/module_adapter/module_adapter.c

```diff
@@ -936,8 +936,11 @@ int module_set_large_config(struct comp_dev *dev, uint32_t param_id, bool first_
 		fragment_size = data_offset_size;
```

Comment:
Minor: why the strange word-wrapping in commit message? 

```
Use correct IPC4 mailbox size. Correct calculation of fragment_size.
```

Nicely fits on one line (with the 74-column typical line-wrap).

Anyways, not blocking PR for this, but just noting this as this seems to repeat in many PRs.

Reply:
@kv2019i  Good point. I'll bare this in mind in the future PRs